style.css

```diff
@@ -0,0 +1,68 @@
+*{
+    margin: 0;
+    padding: 0;
+    box-sizing: border-box;
+}
+body{
+    height: 100vh;
+    display: flex;
+    background-color: rgb(89, 184, 65);
+    transition: background 0.5s ease;
+    position: relative;
+}
+
+.field-name,
+.field-email,
+.field-password,
+.field-finish{
+    position: absolute;
+    top: 50%;
+    left: 50%;
+    transform: translate(-50%, -50%);
+    background: rgb(241, 241, 241);
+    height: 50px;
+    width: 400px;
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    border-radius: 5px;
+    transition: all 0.5 ease;
+}
+.field-name i,
+.field-email i,
+.field-password i,
+.field-finish i{
+    padding: 10px;
+    cursor: pointer;
+}
+.field-name input,
+.field-email input,
+.field-password input{
+    background: none;
+    border: none;
+    flex: 1;
+    height: 100%;
+    outline: none;
+}
+div.innactive{
+    opacity: 0;
+    pointer-events: none;
+    transform: translate(-50%, 50%);
+}
+div.active{
+    opacity: 1;
+    pointer-events: all;
+    transform: translate(-50%, -50%);
+}
+
+@keyframes shake{
+    0%{
+     transform: translate(-50%, -50%) rotate(0deg);
+    }
+    50%{
+        transform: translate(-50%, -50%) rotate(10deg);

```

Comment:
@pravin1237 
What will be the output of the animations? Please do add a screenshot or small video.